packages/components/src/stories/DiagramSequence.stories.js

```diff
@@ -5,6 +5,16 @@ import list_users from '@/stories/data/sequence/list_users.sequence.json';
 import list_users_prefetch from '@/stories/data/sequence/list_users_prefetch.sequence.json';
 import show_user from '@/stories/data/sequence/show_user.sequence.json';
 import user_not_found from '@/stories/data/sequence/user_not_found.sequence.json';
+import { buildStore, SELECT_CODE_OBJECT } from '@/store/vsCode';
+import { Event } from '@appland/models';
+
+const store = buildStore();
```

Comment:
I do want to be moving away from the store as much as possible. It creates an opaque layer between our components and the data they require to render. Maybe this is something that we can drop in the future along with the single observer, N event pattern.

Reply:
I agree and that's probably why prop drilling is best practice. I'm not sure what the alternative is, though. I suppose we could do `provide`/`inject`. Do you think that is less opaque than the store?